src/combinatorics/simple_graph/basic.lean

```diff
@@ -97,21 +106,29 @@ instance complete_graph_adj_decidable (V : Type u) [decidable_eq V] :
   decidable_rel (complete_graph V).adj :=
 λ v w, not.decidable
 
+/-- The graph with no edges on a given vertex type `V`. -/
+def empty_graph (V : Type u) : simple_graph V :=
+{ adj := λ i j, false }
```

Comment:
Should this be a `has_bot` instance?

Reply:
can also be `has_top` with `complete_graph`

Reply:
@eric-wieser @ericrbg Yeah, and `simple_graph` can be a `bounded_lattice`.  Would you mind if we held this off for another PR and merge this one?

Reply:
I've pushed a todo to remind us to do the work.

@hmonroe I think we're free to merge it now

Reply:
bors r+